proto/api.proto

```diff
@@ -1264,7 +1333,7 @@ message GetPricesStreamResponse {
   TokenPrice price = 2;
 }
 
-// Drift messages
```

Comment:
probably either keep `Drift` or delete the comment? `// messages` isn't too helpful